src/api/routes/v1/index.ts

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable import/max-dependencies -- Disabled since this module requires multiple dependencies. */
 import { Router as createRouter } from 'express'
 
+import { handleAmendmentsInfo, handleAmendmentInfo } from './amendments'
```

Comment:
Can these functions have slightly more differentiated names? It's hard to distinguish between the two on first glance.